### PR TITLE
Delete data updates

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/1860

### CODE REVIEW

File: content/influxdb/cloud/write-data/delete-data.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Use the `influx` CLI or the InfluxDB API [`/delete`](/influxdb/v2.0/api/#/paths/~1delete/post) endpoint to delete data.
+
+## Delete data using the influx CLI
+
+{{% note %}}
+If you haven't already, download the [`influx` CLI](/influxdb/cloud/get-started/#optional-download-install-and-use-the-influx-cli).
+{{% /note %}}
+
+1. Use the [`influx delete` command](/influxdb/v2.0/reference/cli/influx/delete/) to delete points from InfluxDB.
+2. Specify your organization, bucket, and authentication token.
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/delete-data.md:60]: - If they're following best practices, they'd have a CLI config that provides the organization and the token. I'm not sure if we need to list them here.
- What about "Use the `-b`, `--bucket` flag to specify which bucket to delete data from."

File: content/influxdb/cloud/write-data/delete-data.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+3. Define the time range to delete data from with the `--start` and `--stop` flags.
+4. (Optional) Specify which points to delete using the predicate parameter and [delete predicate syntax](/influxdb/v2.0/reference/syntax/delete-predicate/).
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/delete-data.md:62]: Are predicate delete's now supported in cloud?

File: content/influxdb/cloud/write-data/delete-data.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Example
+
+```sh
+influx delete -o my-org -b my-bucket -t $INFLUX_TOKEN \
+ --start '1970-01-01T00:00:00.00Z' \
+ --stop '2020-01-01T00:00:00.00Z' \
+ --predicate '_measurement="sensors" and sensorID="abc123"'
+```
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/delete-data.md:71]: I think we've established the convention of always using long-form flags in CLI examples:

```
influx delete --bucket my-bucket \
 --start '1970-01-01T00:00:00.00Z' \
 --stop '2020-01-01T00:00:00.00Z' \
 --predicate '_measurement="sensors" and sensorID="abc123"'
```

File: content/influxdb/cloud/write-data/delete-data.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+## Delete data using the API
+
+1. Use the InfluxDB API `/delete` endpoint to delete points from InfluxDB.
+2. Include your organization and bucket as query parameters in the request URL.
+3. Use the `Authorization` header to provide your InfluxDB authentication token.
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/delete-data.md:77]: your InfluxDB authentication token **with write permissions**.

File: content/influxdb/cloud/write-data/delete-data.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+curl --request POST \
+  https://us-west-2-1.aws.cloud2.influxdata.com/api/v2/delete?orgID=<ORGID> \
+  --header 'Authorization: Token <TOKEN WITH WRITE PERMISSIONS' \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "predicate": "_measurement=\"<MEASUREMENT NAME>\" and _field=\"<FIELD>\"",
+  "start": "2020-08-16T08:00:00Z",
+  "stop": "2020-08-17T08:00:00Z"
+   }'
+```
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/delete-data.md:93]: - Use the `cloud.influxdata.com` placeholder URL
- `bucket` query parameter is missing
- Missing `>` on token placeholder

~~~md
```sh
curl --request POST \
  https://cloud2.influxdata.com/api/v2/delete?org=<org-name>&bucket=<bucket-name> \
  --header 'Authorization: Token <INFLUXDB_AUTH_TOKEN>' \
  --header 'Content-Type: application/json' \
  --data '{
  "predicate": "_measurement=\"example-measurement\" and _field=\"example-field\"",
  "start": "2020-08-16T08:00:00Z",
  "stop": "2020-08-17T08:00:00Z"
   }'
```
~~~

File: content/influxdb/v2.0/write-data/delete-data.md [hunk 6]
```diff
@@ -51,75 +51,31 @@ Delete data from buckets you've created. You cannot delete data from system buck
...
-```
 
-#### Delete data in InfluxDB OSS
+#### Example
 
 ```sh
 influx delete -o my-org -b my-bucket -t $INFLUX_TOKEN \
  --start '1970-01-01T00:00:00.00Z' \
  --stop '2020-01-01T00:00:00.00Z' \
 ```
```
Comment (sanderson) [inline content/influxdb/v2.0/write-data/delete-data.md:69]: Update to use new example conventions:

~~~
```sh
influx delete --bucket my-bucket \
 --start '1970-01-01T00:00:00.00Z' \
 --stop '2020-01-01T00:00:00.00Z' \
```
~~~

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud/write-data/delete-data.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+  --header 'Authorization: Token <INFLUXDB_AUTH_TOKEN>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "predicate": "_measurement=\"example-measurement\" and _field=\"example-field\"",
+  "start": "2020-08-16T08:00:00Z",
+  "stop": "2020-08-17T08:00:00Z"
+   }'
+```
+
+   _For more information, see the [`/delete` API documentation](/influxdb/v2.0/api/#/paths/~1delete/post)._
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/delete-data.md:94]: This doesn't need to be indented.

**Review by sanderson — APPROVED** [review]
One small indentation issue. Looks good!

File: content/influxdb/cloud/write-data/delete-data.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/cloud/write-data/delete-data.md:62]: tested, yes

**Pull request merged**